pyproject.toml

```diff
@@ -61,3 +61,38 @@ version_file = "_icepyx_version.py"
 version_file_template = 'version = "{version}"'
 local_scheme = "node-and-date"
 fallback_version = "unknown"
+
```

Comment:
Using https://docs.astral.sh/ruff/formatter/#docstring-formatting to format the docstrings. This will result in a big diff, so commenting out for now

```suggestion
# [tool.ruff.format]
# docstring-code-format = true
# docstring-code-line-length = "dynamic"
```